.travis.yml

```diff
@@ -5,11 +5,18 @@ php:
 before_script:
 - sudo apt-get update -qq
 - sudo apt-get install -y --force-yes apache2 libapache2-mod-php5 php5-curl php5-mysql php5-intl php5-gd > /dev/null
-- sudo chmod 777 -R /home/travis/build/redCOMPONENT-COM/redCORE/
-- sudo ln -s /home/travis/build/redCOMPONENT-COM/redCORE/tests/ /var/www/tests
+- sudo /etc/init.d/apache2 stop
+- sudo sed -i -e "s,APACHE_RUN_USER=www-data,APACHE_RUN_USER=$USER,g" /etc/apache2/envvars
+- sudo sed -i -e "s,APACHE_RUN_GROUP=www-data,APACHE_RUN_GROUP=$USER,g" /etc/apache2/envvars
+- sudo chown -R $USER /var/lock/apache2
+- sudo chown -R $USER:$USER /var/www
+- ln -s $TRAVIS_BUILD_DIR/tests/ /var/www/tests
 - sudo sed -i -e "s,AllowOverride[ ]None,AllowOverride All,g" /etc/apache2/sites-available/default
-- sudo /etc/init.d/apache2 restart
-- composer install --dev --prefer-dist
+- sudo /etc/init.d/apache2 start
+- if [ ! -f composer.phar ]; then curl -sS https://getcomposer.org/installer | php; fi
+- ./composer.phar install --prefer-dist
+- vendor/phing/phing/bin/phing -f extension_packager.xml autopack
```

Comment:
additionally, I would say that this task needs to be done by RoboFile and not by Travis. Because the idea is that you can execute the tests thanks to Robo no mater if is in travis or in your local machine.
Having this task done in travis it will only affect to the travis build.